tools/debian/convert_debian.py

```diff
@@ -89,33 +93,39 @@ def __repr__(self):
 
 class AdvisoryInfo:
     """Debian advisory info."""
+
     id: str
     summary: str
     details: str
     published: str
     affected: [AffectedInfo]
     aliases: [str]
 
-    def __init__(self, id, summary):
-        self.id = id
+    def __init__(self, adv_id, summary):
+        self.id = adv_id
         self.summary = summary
         self.affected = []
         self.aliases = []
-        self.published = ""
-        self.details = ""
+        self.published = ''
+        self.details = ''
 
     def __repr__(self):
```

Comment:
Let's be explicit and add a `to_dict` method to all of these custom classes. the `__repr__` function can just call that.